### ISSUE TITLE
Make the bot more conversational

### ISSUE THREAD

**Issue by evijit** [AvijitGhosh82/sheldonisms#2]
Api.ai has some fixed replies, but we must make sure the bot has character

**Comment by parinaya-007** [comment]
so do we need to modify the api.ai to make the chatbot more conversational ?

**Comment by evijit** [comment]
Yes. Please paste a list of the changes you are making there so that I can see and comment if required.
